roqoqo-qasm/src/interface.rs

```diff
@@ -154,6 +193,32 @@ pub fn call_operation(
                 op.qubit()
             ))
         }
+        Operation::PragmaActiveReset(op) => {
+            Ok(format!("reset {}[{}];", qubit_register_name, op.qubit(),))
+        }
+        Operation::PragmaConditional(op) => {
+            // can't handle multiple operations under if condition
+            let mut ite = op.circuit().iter().peekable();
+            let mut data = "".to_string();
+            while let Some(int_op) = ite.next() {
+                if ite.peek().is_none() {
+                    data.push_str(&format!(
+                        "if({}[{}]==1) {}",
```

Comment:
Should this end with a semi-colon?

Reply:
I'm double checking, but the internal call_operation(...) should handle that.